common.gypi

```diff
@@ -36,7 +36,7 @@
 
     # Reset this number to 0 on major V8 upgrades.
     # Increment by one for each non-official patch applied to deps/v8.
-    'v8_embedder_string': '-node.18',
+    'v8_embedder_string': '-node.13',
```

Comment:
Is this correct?

Reply:
There are 13 floating patches on V8, so yes.